### PR TITLE
delete comment functionality added

### PR DESCRIPTION
**Pull request by tashkenna**

### CODE REVIEW

File: src/components-pages/Login.jsx [hunk 1]
```diff
@@ -1,13 +1,61 @@
...
+import { UserCard } from "../components/UserCard";
+
 
 export const Login = () => {
-    const [user, setUser] = useState("")
+  const [users, setUsers] = useState([]);
+  const { setUser, user } = useContext(UserContext);
+
+  useEffect(() => {
+    getUsers().then(({ users }) => {
```
Comment (popatre) [inline src/components-pages/Login.jsx:14]: Need some loading for this, otherwise it looks like the screenshot/broken
Screenshot 2023-07-21 at 10 22 40

**Review by popatre — CHANGES_REQUESTED** [review]
- Nice feedback/updating of the UI when a comment gets deleted. 
- Nice handling of the user being able to only delete their own comments
- Excellent not even having the option/button to delete if not logged in

- [x] You've got your error state/message, however it appears at the top. Its more a UX thing, but if i try and delete a comment at the bottom, whilst offline, i dont know anything happened because the message is at the top. Ideally the message would be in the card, you could also look at this little package which will disable a message nicely - https://react-hot-toast.com/

- [x] Would be good if you disabled the delete button after clicking so they can't click it multiple times to delete the same comment - on a slow connection, i can keep clicking the button because it looks like nothing is happening. Would be good for it to be disabled whilst deleting. Ideally, the delete message would also be in the card, instead of at the top.

**Review by MKD01 — CHANGES_REQUESTED** [review]
Im a big fan of this library 👍

However when a delete request fails the error message does not show up, and it instead shows me the successful message of comment deleted 🤔. 

Even though your delete button is actually disabled while the request is going through, visually there is no indication of this, so even though my clicks are not doing anything I can still keep clicking the button as if it was working. As a user It would be nice to be able to know the button has been disabled at a glance as well.

You're almost there with this one, once you have sorted the above please resubmit this pr 👍

**Review by MKD01 — APPROVED** [review]
Great work im happy for you to merge this in 👍

**Pull request merged**